### PR TITLE
Updated font installation documentation

### PR DESCRIPTION
**Pull request by 0ctagon**
The script to install the fonts is a bit more complex compared to the time we wrote the troubleshooting instructions. It is now updated to reflect what is going on.
Also added a small check to not download the fonts if they are already there.

See doc here: https://plothist.readthedocs.io/en/install-font-doc-update/usage/font_installation.html

### CODE REVIEW

File: docs/usage/font_installation.rst [hunk 1]
```diff
@@ -33,17 +36,31 @@ Then you can run the following command in a terminal:
...
+- `Latin Modern Math <http://mirrors.ctan.org/fonts/lm-math/opentype/latinmodern-math.otf>`_
+- `Latin Modern Roman <https://www.1001fonts.com/download/latin-modern-roman.zip>`_
+- `Latin Modern Sans <https://www.1001fonts.com/download/latin-modern-sans.zip>`_
 
-   font_manager.findSystemFonts(fontpaths=None, fontext="ttf")
+After downloading the fonts, they should be placed in the folder ``~/.fonts/``. If the folder doesn't exist, you can create it by running in a terminal:
+
+.. code-block:: bash
+
+   mkdir -p ~/.fonts
```
Comment (cyrraz) [inline docs/usage/font_installation.rst:53]: Why -p? I would remove it

File: docs/usage/font_installation.rst [hunk 2]
```diff
@@ -33,17 +36,31 @@ Then you can run the following command in a terminal:
...
+
+Then you can move the downloaded fonts to the folder by running in a terminal:
+
+.. code-block:: bash
+
+   mv ~/path/to/latinmodern-math.otf ~/.fonts/
```
Comment (cyrraz) [inline docs/usage/font_installation.rst:59]: Remove ~/ at the beginning of the path to downloaded fonts

File: docs/usage/font_installation.rst [hunk 3]
```diff
@@ -33,17 +36,31 @@ Then you can run the following command in a terminal:
...
+   unzip -o ~/path/to/latin-modern-roman.zip -d ~/.fonts/latin-modern-roman
```
Comment (cyrraz) [inline docs/usage/font_installation.rst:60]: Remove ~/ at the beginning of the path to downloaded fonts

File: docs/usage/font_installation.rst [hunk 4]
```diff
@@ -33,17 +36,31 @@ Then you can run the following command in a terminal:
...
+   unzip -o ~/path/to/latin-modern-sans.zip -d ~/.fonts/latin-modern-sans
```
Comment (cyrraz) [inline docs/usage/font_installation.rst:61]: Remove ~/ at the beginning of the path to downloaded fonts

File: docs/usage/font_installation.rst [hunk 5]
```diff
@@ -52,3 +69,18 @@ Make also sure to delete the cache of matplotlib, otherwise the fonts may not be
 
    cache_dir = matplotlib.get_cachedir()
    subprocess.run(["rm", "-rv", cache_dir])
+
+Try to restart your python session and run ``import plothist`` again. If no warning is raised, the fonts are installed correctly and you can skip the rest of this section.
```
Comment (cyrraz) [inline docs/usage/font_installation.rst:73]: I would remove this sentence, it is already present at the beginning of the page.

**Review by cyrraz — CHANGES_REQUESTED** [review]
Thank you, just a couple of minor comments.

**Comment by 0ctagon** [comment]
I re-organized the page, placing `fonts not installed troubleshouting` first

**Pull request merged**